Sming/Components/crypto/include/Crypto/Md5.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
+
+#include "HashContext.h"
+
+#ifdef ARCH_ESP8266
+#include <esp_crypto.h>
+#define CRYPTO_DEF(f) ESP_MD5_##f
+#else
+#define CRYPTO_DEF(f) MD5_##f
```

Comment:
The `#else` case should also include a header where the `MD5_` functions are defined.